src/rules/no-ignored-unsubscribe.ts

```diff
@@ -0,0 +1,30 @@
+import type { TSESTree } from '@typescript-eslint/types';
+import { createRule } from "../utils";
+
+export default createRule("no-ignored-unsubscribe", {
+	meta: {
+		docs: {
+			description: "Forbids ignoring the unsubscribe method returned by the `subscribe()` os Svelte stores.",
```

Comment:
Start with `disallow` according to the documentation. Also, there's probably a typo.

```suggestion
			description: "disallow ignoring the unsubscribe method returned by the `subscribe()` on Svelte stores",
```